.github/workflows/image.yml

```diff
@@ -47,69 +34,14 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and push by digest
+      - name: Build and push
         id: build
         uses: docker/build-push-action@16ebe778df0e7752d2cfcbd924afdbbd89c1a755 # v6.6.1
         with:
           context: .
+          provenance: mode=max
+          sbom: true
           push: ${{ startsWith(github.ref, 'refs/tags') }}
```

Comment:
As this workflow is triggered only on `release` event I think `push: true` is enough but doesn't hurt to check ref.

We should also build on PR to make sure the workflow runs fine imo:

```yaml
on:
  release:
    types: [published]
  pull_request:
permissions:
  packages: write
```

Then `push` can still be:

```yaml
push: ${{ startsWith(github.ref, 'refs/tags') }}
```

Or 

```yaml
push: ${{ github.event_name != 'pull_request' }}
```

Can be a follow-up if you want